ts/features/itwallet/common/store/reducers/index.ts

```diff
@@ -43,15 +45,28 @@ const itwReducer = combineReducers({
   preferences: preferencesReducer
 });
 
-const CURRENT_REDUX_ITW_STORE_VERSION = 1;
+const CURRENT_REDUX_ITW_STORE_VERSION = 2;
 
 const migrations: MigrationManifest = {
   // Added preferences store
   "0": (state: PersistedState): PersistedState =>
     _.set(state, "preferences", {}),
+
   // Added requestedCredentials to preferences store
   "1": (state: PersistedState): PersistedState =>
-    _.set(state, "preferences.requestedCredentials", {})
+    _.set(state, "preferences.requestedCredentials", {}),
+
+  // Added authLevel to preferences store and set it to "L2" if eid is present
+  "2": (state: PersistedState): PersistedState => {
+    const eid: O.Option<StoredCredential> = _.get(state, "credentials.eid");
```

Comment:
Why do we need to use `lodash.get`?

Reply:
Thank you for pointing out that it is not necessary to use `lodash`, I responded to you in your other comment about that as well with a suggested refactor.